Core/Core/CourseSync/CourseSyncDownloader/CourseSyncDownloaderAssembly.swift

```diff
@@ -42,7 +42,7 @@ public enum CourseSyncDownloaderAssembly {
         return CourseSyncInteractorLive(contentInteractors: contentInteractors,
                                         filesInteractor: CourseSyncFilesInteractorLive(),
                                         progressWriterInteractor: CourseSyncProgressWriterInteractorLive(),
-                                        successNotification: CourseSyncSuccessNotificationInteractor(notificationManager: .shared,
+                                        notification: CourseSyncNotificationInteractor(notificationManager: .shared,
```

Comment:
I'd probably rename this to `notificationInteractor` so it's more consistent with the other property namings.

Reply:
Agree, renamed.